src/couch/src/couch_debug.erl

```diff
@@ -18,6 +18,18 @@
     opened_files_contains/1
 ]).
 
+-export([
+    process_name/1,
+    link_tree/1,
+    link_tree/2,
+    mapfold/3,
```

Comment:
Printing list of processes is nice but if you have thousands of processes you would want to get the tree and operate on it (filter or transform) directly. Here is where the map/mapfold/fold are handy.